docs/tutorials/jsx-transform-migration.md

```diff
@@ -0,0 +1,477 @@
+---
+id: jsx-transform-migration
+title: Transitioning to the New JSX Transform
+description: A guide to migrating your project to the New JSX Transform
+---
+
+The Backstage core libraries are in the process of deprecating React 16 and evaluating the adoption of React 19. In React 19, the [New JSX Transform](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html), introduced in September 2020, is now [mandatory](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#installing). This requires a modification in how React is imported into components to ensure compatibility with React 19.
+
+## Migration Process
+
+### Using the Codemod
+
+The codemod referenced in the [Introducing the New JSX Transform](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) article is no longer functional. However, we have identified a working solution detailed below.
```

Comment:
Should we mention this if it doesn't work anyway? 💭

Reply:
You're right I think its fine as long as it mentions to not use it because its not working 😅